### ISSUE TITLE
LED indicators for vision targets

### ISSUE THREAD

**Issue by richiksc** [TheGreenMachine/Zeta#39]
Current suggestion: Magenta (#ff00ff / 255 0 255) as indicator for vision target located.
Alternative colors: Cyan (#00ffff / 0 255 255). The color must be visibly different enough from colors currently in use but can not look like an "error" or "warning" color.

Backed by @kooryan, @ianmcvann, and @richiksc.

**Comment by andrewfhou** [comment]
As cyan is still a 'blue', I think magenta would be preferable, since we use blue to indicate endgame state. 

This should be fairly trivial to implement, we just need a good way to remember state so it isn't overridden by the default teleop green LED state (but is overridden by the endgame blue LED state)

**Comment by andrewfhou** [comment]
Would you want this indicator to be active at all times during a match, or only when the vision navigation commands are being run?

**Comment by richiksc** [comment]
Only when the vision nav commands are being run. I do know @kooryan wanted to work on this as he let me know personally.

**Comment by andrewfhou** [comment]
@richiksc @kooryan Got it. I think this is lower priority than tuning the system itself, but is easy to work on in parallel, so we'll tag as such and put it on Saturday's docket.

And yes, I agree - I was initially under the impression that the indicator state would be active at all times, not just when the command is being run. 666bcfc adds the referenced color, and adds comment labels for the various state colors, didn't see your comment earlier.

Note that I think the easiest way to do state control for preventing conflicting LED state commands is to check the currently running command for drivetrain (using the `isRunning()` method in the command class)
